guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -187,17 +189,19 @@ Contributing to the Rails Code
 
 To move on from submitting bugs to helping resolve existing issues or contributing your own code to Ruby on Rails, you _must_ be able to run its test suite. In this section of the guide, you'll learn how to set up the tests on your computer.
 
-#### The Easiest Way
+#### Using Codespaces
+
+Start a GitHub Codespace from the repository and start developing right away from the browser IDE or in your local VS Code. The Codespace is initialized with all required dependencies and allows you to run all tests.
```

Comment:
I have a feeling this only works if your user account is part of an organisation that has codespaces enabled. So it won't work for everyone. See https://github.com/rails/rails/pull/43061#issuecomment-924152550

![image](https://user-images.githubusercontent.com/509837/158650889-fb44c0d9-5324-4b4c-be10-8a237d976fe1.png)

Reply:
Alright yea I totally forgot about the limitations of codespaces 😭 

I've updated this to say if you have a codespace enabled org, fork into that org and use the codespace there. Otherwise you can use the devcontainer file with docker and the remote vscode plugin.